lib/discovery.dart

```diff
@@ -5,9 +5,9 @@
 library package_config.discovery;
 
 import "dart:async";
-import "dart:io" show Directory, File, FileSystemEntity;
+import "dart:io";
+import "dart:typed_data" show Uint8List;
 import "package:path/path.dart" as path;
-import "package:http/http.dart" as http;
 import "packages.dart";
 import "packages_file.dart" as pkgfile show parse;
```

Comment:
Nit: while we're here, consider adding a newline separator here as per the [style guide recs](https://www.dartlang.org/articles/style-guide/#ordering).